### ISSUE TITLE
BanIPs-CF

### ISSUE THREAD

**Issue by BroadRippleInc** [andrewsalmon/BanIPs-CF#1]
I am not able to make this work. Are you available for hire? This is exactly the script I am looking for, any help is appreciated and I'm happy to pay for your time.

If I unban I get "You disabled saving to file; Cannot continue!"

If I Input NGINX Logs the confirm button doesn't do anything.

If I ban IP, the page reloads with no message.

Thanks in advance.

**Comment by dehlirious** [comment]
Yeah, I am. And unfortunately Cloudflare made some sort of change that broke it, I'll have to give the script another shot. 
I'm terribly sorry for the delayed response. It's been an adventure as of late.

Of course, I'm here to assist. I've noticed that Cloudflare's recent updates seem to have impacted the functionality of the script. I'll thoroughly review and address it. I apologize for not getting back to you sooner. It's been a whirlwind for me, but I'm committed to assisting you in one form or another.
